f8a_worker/workers/mercator.py

```diff
@@ -271,3 +275,22 @@ def get_outermost_items(list_):
             sorted_list = [i for i in sorted_list if
                            len(i['path'].split(os.path.sep)) == outermost_len]
         return sorted_list
+
+
+def validate_utf_json(result):
```

Comment:
This can be a private function as nobody else accesses it from outside.

Reply:
mercator is used for the pkg analysis. Its not used for stack reports